SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -86,22 +86,31 @@ Worker_ComponentData CreateGlobalStateManagerData()
 
 Worker_ComponentData CreateDeploymentData()
 {
-	// Construct the Deployment component data object.
 	Worker_ComponentData DeploymentData;
 	DeploymentData.component_id = SpatialConstants::DEPLOYMENT_MAP_COMPONENT_ID;
 	DeploymentData.schema_type = Schema_CreateComponentData(SpatialConstants::DEPLOYMENT_MAP_COMPONENT_ID);
 	Schema_Object* DeploymentDataObject = Schema_GetComponentDataFields(DeploymentData.schema_type);
 
-	// Add the MapURL schema field.
 	Schema_Object* MapURLObject = Schema_AddObject(DeploymentDataObject, SpatialConstants::GLOBAL_STATE_MANAGER_MAP_URL_ID);
 	AddStringToSchema(MapURLObject, 1, TEXT("default")); // TODO: Fill this with the map name of the map the snapshot is being generated for.
 
-	// Add the accepting players schema field.
 	Schema_AddBool(DeploymentDataObject, SpatialConstants::GLOBAL_STATE_MANAGER_ACCEPTING_PLAYERS_ID, false);
 
 	return DeploymentData;
 }
 
+Worker_ComponentData CreateStartupActorManagerData()
+{
+	Worker_ComponentData StartupActorManagerData;
```

Comment:
All the Worker_ComponentDatas in this file should have zero initialization applied - {}

Reply:
it's called uniform initialization. Mike please